schema/private.openapi.yaml

```diff
@@ -127,6 +127,8 @@ components:
       properties:
         recipient:
           $ref: './public.openapi.yaml#/components/schemas/RunRecipient'
+        org_id:
```

Comment:
We should not be changing v1 of the API - whomever needs to use org_id should move to v2.